scope.go

```diff
@@ -469,7 +473,7 @@ func (scope *Scope) scan(rows *sql.Rows, columns []string, fields []*Field) {
 		}
 
 		for fieldIndex, field := range selectFields {
-			if field.DBName == column {
+			if strings.EqualFold(field.DBName, column) {
```

Comment:
Case insensitive comparison: This is necessary because Oracle returns the column names in UPPER CASE.